src/neuroglancer/widget/segment_set_widget.ts

```diff
@@ -224,10 +262,21 @@ export class SegmentSetWidget extends RefCounted {
         itemButton.addEventListener('mouseenter', function(this: HTMLButtonElement) {
           temp.tryParseString(this.textContent!);
           widget.segmentSelectionState.set(temp);
+          widget.segmentSelectionState.setRaw(temp);
+          const segmentButton =
+              <HTMLElement>document.querySelector(`[data-segID="${temp.toString()}"]`);
+          segmentButton.className += ' selectedSeg';
```

Comment:
`segmentButton.className.add('selectedSeg');` also takes care of duplicate class assignment, and doesn't require the whitespace